web/controllers/game_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Yggdrasil.GameController do
+  use Yggdrasil.Web, :controller
+
+  alias Yggdrasil.Game
+
+  def index(conn, _params) do
+    games = Repo.all Game
```

Comment:
@copenhas in-light of the changes you did with splitting the app, I don't think I want to do this in the controller.

I'm going to have to do some clean up and reorganize anyway with this change now in master so do you have maybe some suggestions on how we cleanly present these sorts of things to the web layer?



```Exlir
# Almost in favor of having Game
Yggdrasil.Game.list_games()
Yggdrasil.Game.get_game(123)
Yggdrasil.Game.list_characters()
Yggdrasil.Game.get_character(123)
Yggdrasil.Game.add_character(...)
```

We do have a game module for the game struct, but either we can call that something like GameModel or use module names to differentiate (not sure how I feel about that yet, but may be an option) so like `Yggdrasil.Model.Game` for the game model, and `Yggdrasil.Game` for the api exposed to the web... Again GameModel maybe a better a choice there but suggesting both here.